src/main/java/frc/robot/subsystems/HangSubsystem.java

```diff
@@ -52,16 +68,28 @@ public void liftToHeight() {
     }
   }
 
+  public double liftPosition() {
+    return liftEncoder.getPosition();
+  }
+
   public void winchAndLowerLift() {
     // liftMotor.set(-0.42);
     // winchMotor.set(1);
-    if (Math.abs(liftEncoder.getPosition()) > 1) {
-      liftMotor.set(-HangConstants.LIFT_SPEED);
-      winchMotor.set(1);
-    } else {
-      liftMotor.stopMotor();
-      winchMotor.stopMotor();
-    }
+    // if (Math.abs(liftEncoder.getPosition()) > 1) {
+    liftMotor.set(-HangConstants.LIFT_SPEED);
+    // winchMotor.set(1);
+    // } else {
+    // liftMotor.stopMotor();
+    // winchMotor.stopMotor();
+    // }
+  }
+
+  public void winchStart() {
+    winchMotor.set(.5);
```

Comment:
Let's use a constant here:

```suggestion
    winchMotor.set(HangConstants.WINCH_MOTOR_SPEED);
```

or something